lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -481,12 +497,32 @@ def rules_expand_sources(rules):
             for rule in rule_expand_sources(rule_complex)]
 
 
+def update_rules_description(module, client, rule_type, group_id, ip_permissions):
+    try:
+        if rule_type == "in":
+            client.update_security_group_rule_descriptions_ingress(GroupId=group_id, IpPermissions=[ip_permissions])
```

Comment:
This could presumably fail with AttributeError if a user hasn't recently updated botocore.

Reply:
Thanks, I have addressed this in 7061e623ca34799693f213a7625bc19e1ac5681b. There might be a more elegant solution though, and I'm not sure if I should be making it a global - perhaps I should be passing it around explicitly. Let me know if you have a preference please.

I have also added to ec2-policy.json.